UltiSnips/nix.snippets

```diff
@@ -0,0 +1,38 @@
+snippet sha256 "sha256"
+sha256 = "0000000000000000000000000000000000000000000000000000";
+endsnippet
+
+snippet fetchFromGitHub "fetchFromGitHub"
+fetchFromGitHub {
+  owner = "${1}";
+  repo = "${2}";
+  rev = "${3:v\${version}}";
+  sha256 = "0000000000000000000000000000000000000000000000000000";
+}
+endsnippet
+
+snippet fetchurl "fetchurl"
+fetchurl {
+  url = "${1}";
+  sha256 = "0000000000000000000000000000000000000000000000000000";
+}
+endsnippet
+
+snippet stdenv "stdenv.mkDerivation"
+stdenv.mkDerivation rec {
+  name = "${1}";
```

Comment:
```
snippet meta "meta"
meta = with stdenv.lib; {
   description = "${1}";
   homepage = "${2}";
   license = licenses.${3};
   maintainers = with maintainers; [ ${4} ];
};
endsnippet
```

Reply:
I think pname/version is the de-facto standard these days, so this may be more useful.